packages/core/upload/admin/src/hooks/tests/useModalQueryParams.test.ts

```diff
@@ -63,7 +63,7 @@ describe('useModalQueryParams', () => {
         ...FIXTURE_QUERY.filters,
         $and: [
           {
-            some: 'thing',
+            some: { $eq: 'thing' },
```

Comment:
Same question as in your other PR, how come we have actual changes in the tests? in this case why did it pass before?

Reply:
yes, you are right, I am gonna replace the old code